coordinator/src/main/scala/filodb.coordinator/ResultActor.scala

```diff
@@ -0,0 +1,33 @@
+package filodb.coordinator
+
+import akka.actor.Props
+import monix.reactive.MulticastStrategy
+import monix.reactive.subjects.ConcurrentSubject
+
+import filodb.coordinator.ActorSystemHolder.system
+import filodb.query.Query.qLogger
+import filodb.query.StreamQueryResponse
+
+
+object ResultActor {
+  def props(subject: ConcurrentSubject[StreamQueryResponse, StreamQueryResponse]): Props =
+    Props(classOf[ResultActor], subject)
+  lazy val subject = ConcurrentSubject[StreamQueryResponse](MulticastStrategy.Publish)(QueryScheduler.queryScheduler)
+  lazy val resultActor = system.actorOf(Props(new ResultActor(subject)))
+}
+
+class ResultActor(subject: ConcurrentSubject[StreamQueryResponse, StreamQueryResponse]) extends BaseActor {
+
+  def receive: Receive = {
+    case q: StreamQueryResponse =>
+      try {
+        subject.onNext(q)
+        qLogger.debug(s"Result Actor got ${q.getClass} as response from ${sender()}")
+      } catch {
+        case e: Throwable =>
+          qLogger.error(s"Exception when processing $q", e)
```

Comment:
onNext typically should not throw exception. It would instead cause observable to fail and that path would kick in. I do not want to corrupt that path by onNexting another error.